yt_dlp/extractor/cbc.py

```diff
@@ -228,6 +229,63 @@ def _real_extract(self, url):
         }
 
 
+class CBCPlayerPlaylistIE(InfoExtractor):
+    IE_NAME = 'cbc.ca:player:playlist'
+    _VALID_URL = r'(?:cbcplayer:|https?://(?:www\.)?cbc\.ca/(?:player/)(?!play/))(?P<id>.+)'
+    _TESTS = [{
+        'url': 'https://www.cbc.ca/player/news/TV%20Shows/The%20National/Latest%20Broadcast',
+        'playlist_mincount': 25,
+        'info_dict': {
+            'id': 'news/tv shows/the national/latest broadcast',
+            'ie_key': 'CBCPlayer',
+        }
+    }, {
+        'url': 'https://www.cbc.ca/player/news/Canada/North',
+        'playlist_mincount': 25,
+        'info_dict': {
+            'id': 'news/canada/north',
+            'ie_key': 'CBCPlayer',
+        }
+    }]
+
+    def _real_extract(self, url):
+        # We have no other playlist id other than the URL, so use that
+        playlist_id = self._match_id(url).replace('%20', ' ').lower()
```

Comment:
IMO use `urllib.parse.unqouote` (and add `import urllib.parse` to top of file) in case there is other percent-encoding in URL
```suggestion
        playlist_id = urllib.parse.unquote(self._match_id(url)).lower()
```